js/rpg_core/Sprite.js

```diff
@@ -222,10 +227,16 @@ Sprite.prototype.setColorTone = function(tone) {
  * @private
  */
 Sprite.prototype._onBitmapLoad = function() {
-    if (this._frame.width === 0 && this._frame.height === 0) {
-        this._frame.width = this._bitmap.width;
-        this._frame.height = this._bitmap.height;
+    if(this._requestBitmap === this._bitmap){
```

Comment:
I don't know why `this._requestBitmap` is defined, because it's always the same as `this._bitmap`.